server/src/com/vizdom/dbd/jdbc/Connection.java

```diff
@@ -419,14 +419,15 @@ BerObject handleRequest(ConnectRequest aRequest)
                 else
                     mConn = DriverManager.getConnection(mUrl, mUser, mPassword);
 
-                DatabaseMetaData dbmd = mConn.getMetaData(); 
-                gLog.debug("Created database connection to " + 
-                    dbmd.getDatabaseProductName() + " v" + 
-                    dbmd.getDatabaseProductVersion()); 
-                mSupportsGetGeneratedKeys = dbmd.supportsGetGeneratedKeys(); 
+                DatabaseMetaData dbmd = mConn.getMetaData();
+                gLog.debug("Created database connection to " +
+                    dbmd.getDatabaseProductName() + " v" +
+                    dbmd.getDatabaseProductVersion());
+                //mSupportsGetGeneratedKeys = dbmd.supportsGetGeneratedKeys();
```

Comment:
We disabled that one because it causes some issues with MARS in SQL Server 2014+
It should be a "property" to enable/disable the detection.